src/main/java/edu/kis/powp/jobs2d/features/DriverFeatureChangeObserver.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.observer.Subscriber;
+
+public class DriverFeatureChangeObserver implements Subscriber {
```

Comment:
The name should primarily indicate the behavior (what does DriverFeature.updateDriverInfo() do?)